src/midi.js

```diff
@@ -5,6 +5,68 @@ const assert = require('assert');
 const jsmidgen = require('jsmidgen');
 const setMiddleC = require('./setMiddleC');
 
+function doesArrayContainAnArray(arr){
+  return arr.some(function(ele){
+    return Object.prototype.toString.call( ele ) === '[object Array]'
+  });
+}
```

Comment:
We can use `Array.isArray` and convert it to a single line:
```
const doesArrayContainAnArray = arr => arr.some(ele => Array.isArray(ele));
```

Also we can move it to utils.js and make it available from there so that it may be available to other files as well. Please add tests for it as well.